weldx/tests/asdf_tests/test_weldx_file.py

```diff
@@ -377,7 +377,7 @@ def get_mem_info():
             diff = after - before
             # pytest increases memory a bit, but not as much as our large array would
             # occupy in memory.
-            assert diff <= large_array.nbytes * 1.1, diff / 1024 ** 2
+            assert diff <= large_array.nbytes * 1.1, diff / 1024**2
```

Comment:
eventually it is more readable, but I liked the consistency of the "spaces between operators" rule more.

Reply:
yes that change also surprised me, alas...